workspaces/arborist/test/fixtures/file-dep/fake/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "bundleDependencies": true,
+  "dependencies": {
+    "abbrev": "2.0.0"
+  },
+  "devDependencies": {
+    "has-package-exports": "file:."
```

Comment:
The package.json in my “fake” fixture was originally copied out of a node_modules directory, produced by an npm install at the time (which version, i have no idea).

Reply:
ah, and then in the repo in question, i do `git ls-files fixtures/*/project/package.json | cut -d / -f 2 | xargs -I {} echo @fixtures/{}@file:./fixtures/{}/project | xargs npm install --save-dev`.

Perhaps though one important piece is that `package-lock=false` in `.npmrc` - meaning, there's never an actual lockfile to work with.

Reply:
That could be a detail of this. I removed it from the test case because by default the fixture has no package-lock so the reify won’t use one. One will be created but then is immediately cleaned up by tap, so it has no effect on the test.

Reply:
I tried with `package-lock=false` and got the same failure. It looks like `treeCheck()` gets the resolved tree data structure after reification is complete, so in this case that is without a package-lock.json file.